sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1027,9 +1027,12 @@ def test_ridge_fit_intercept_sparse():
         assert_array_almost_equal(dense.coef_, sparse.coef_)
 
     # test the solver switch and the corresponding warning
-    for solver in ['saga', 'lsqr']:
-        sparse = Ridge(alpha=1., tol=1.e-15, solver=solver, fit_intercept=True)
-        assert_raises_regex(ValueError, "In Ridge,", sparse.fit, X_csr, y)
+    for solver in ['saga', 'lsqr', 'sag']:
+        sparse = Ridge(alpha=1., solver=solver, fit_intercept=True)
```

Comment:
Avoid to call it `sparse`. I think that we have the following import sometimes: `from scipy import sparse`.

Reply:
+1